ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -315,11 +350,26 @@ std::vector<std::unique_ptr<SingleProcessData>> createPerProcessData(
 
 std::unique_ptr<UncoupledProcessesTimeLoop> createUncoupledProcessesTimeLoop(
     BaseLib::ConfigTree const& config, std::string const& output_directory,
-    const std::map<std::string, std::unique_ptr<Process>>&
-        processes,
+    const std::map<std::string, std::unique_ptr<Process>>& processes,
     const std::map<std::string, std::unique_ptr<NumLib::NonlinearSolverBase>>&
         nonlinear_solvers)
 {
+    auto const& coupling_config
+        //! \ogs_file_param{prj__time_loop__global_process_coupling}
+        = config.getConfigSubtreeOptional("global_process_coupling");
+
+    std::unique_ptr<NumLib::ConvergenceCriterion> coupling_conv_crit = nullptr;
+    unsigned max_coupling_iterations = 1;
+    if (coupling_config)
+    {
+        max_coupling_iterations
+            //! \ogs_file_param{prj__time_loop__global_process_coupling__max_iteration}
+            = coupling_config->getConfigParameter<unsigned>("max_iteration");
```

Comment:
How about change ``max_iter`` in  nonlinear solver to ``max_iteration``? If you agree this, I will change it in another PR.  Otherwise, I will change ``max_iteration`` to ``max_iter``.